Examples/DicomConvert/DicomConvert.py

```diff
@@ -0,0 +1,110 @@
+#=========================================================================
+#
+#  Copyright NumFOCUS
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#=========================================================================
+
+import SimpleITK as sitk
+import multiprocessing
+import argparse
+import os
+import sys
+import functools
+import pandas as pd
```

Comment:
Please don't use pandas in examples. We don't want the additional dependency.

Reply:
Done.